### PR TITLE
Feat/up ceramic ipfs 54 test await

### PR DESCRIPTION
**Pull request by zachferland**

### CODE REVIEW

File: test/threeid-provider.test.js [hunk 1]
```diff
@@ -65,7 +70,7 @@ describe('ThreeIdProvider', () => {
   beforeAll(async () => {
     tmpFolder = await tmp.dir({ unsafeCleanup: true })
     ipfs = await Ipfs.create(genIpfsConf(tmpFolder.path))
-    ceramic = await Ceramic.create(ipfs, { stateStorePath: tmpFolder.path + '/ceramic/' })
+    ceramic = await Ceramic.create(ipfs, { stateStoreDirectory: tmpFolder.path + '/ceramic/', anchorOnRequest: false })
```
Comment (oed) [inline test/threeid-provider.test.js:73]: Not needed, but we could just use the `jest-environment-ceramic` instead of manually setting up a node here.

**Review by oed — APPROVED** [review]
LGTM, just one potential improvement possible.

File: test/threeid-provider.test.js (hunk 1 above)
Comment (oed) [reply test/threeid-provider.test.js:73]: See https://github.com/ceramicstudio/js-idx/blob/master/packages/tools/test/lib.test.ts#L1-L3

Comment (zachferland) [reply test/threeid-provider.test.js:73]: oh nice, did know we had that, but yeah will add as part of future cleanup story

**Pull request merged**